service/src/main/java/bio/terra/pipelines/dependencies/rawls/RawlsService.java

```diff
@@ -314,4 +315,9 @@ static <T> T executionWithRetryTemplate(RetryTemplate retryTemplate, RawlsAction
           }
         });
   }
+
+  public static String createDataTableEntityName(
```

Comment:
take or leave: in case we want to change this at all in future, what do you think of making the input just a Pipeline object and extracting the name and version out of it inside the method?
edit: after seeing the rest of the PR, i think the answer is that we have pipelineName and (now) pipelineVersion in the working map (input params), and not the Pipeline object. that makes sense. we could consider a refactor to just have the Pipeline object in the working map but doesn't make sense to do it here.
all to say, looks great!

Reply:
yea i had the same thought process you put into text here 😂 . I do think refactoring will be worth doing if we continue adding pipeline fields to flights